Jenkinsfile.open.si

```diff
@@ -1,45 +1,47 @@
 #!/usr/bin/env groovy
 
-node('py36') {
-  wrap([$class: 'MesosSingleUseSlave']) {
-  wrap([$class: 'AnsiColorBuildWrapper', 'colorMapName': 'XTerm']) {
+timeout(time: 3, unit: 'HOURS') {
+    node('py36') {
```

Comment:
Nit: I'd avoid the indented code to ease reviewing.

Reply:
You're right, but that's Intellij re-formating everything. I decided not to argue with it 😩

Reply:
@jeschkies this can help!

![image](https://user-images.githubusercontent.com/1043441/49591402-ad0f9b00-f96e-11e8-8753-32e8cb5ea013.png)

Reply:
Oh man. Thanks a ton!